draft-ietf-quic-tls.md

```diff
@@ -667,73 +600,85 @@ Early data within the TLS connection MUST NOT be used.  As it is for other TLS
 application data, a server MUST treat receiving early data on the TLS connection
 as a connection error of type PROTOCOL_VIOLATION.
 
+## Rejecting 0-RTT
 
-## QUIC Key Expansion {#key-expansion}
+A server rejects 0-RTT by rejecting 0-RTT at the TLS layer.  This results in
+early exporter keys being unavailable, thereby preventing the use of 0-RTT for
+QUIC.
 
-QUIC uses a system of packet protection secrets, keys and IVs that are modelled
-on the system used in TLS {{!TLS13}}.  The secrets that QUIC uses
-as the basis of its key schedule are obtained using TLS exporters (see Section
-7.5 of {{!TLS13}}).
+A client that attempts 0-RTT MUST also consider 0-RTT to be rejected if it
+receives a Version Negotiation packet.
```

Comment:
Or a Retry?  While I'm not clear on the reasons, QUIC currently prohibits 0-RTT after a Retry packet.  If the reasons are still valid, that should be mentioned here; if not, we need to make sure that text is removed by this PR.

Reply:
It's no longer relevant. Do you know where it says that?